api/auth/approle/approle.go

```diff
@@ -22,13 +22,12 @@ type AppRoleAuth struct {
 var _ api.AuthMethod = (*AppRoleAuth)(nil)
 
 // SecretID is a struct that allows you to specify where your application is
-// storing the secret ID required for login to the AppRole auth method.
+// storing the secret ID required for login to the AppRole auth method. The
+// recommended secure pattern is to use response-wrapping tokens rather than
```

Comment:
Good idea moving the recommendation here where it's more easily discoverable.

Reply:
I figured the recommendation now applies to all 3 options so it makes sense to move it up.